### PR TITLE
Complete assignment

### PR DESCRIPTION
**Pull request by ash827**

### CODE REVIEW

**Comment by llpk79** [comment]
# Data Science Unit 4 Sprint 2 Assignment 2

### Implement a 3 input, 4 node hidden-layer, 1 output node Multilayer Perceptron on the following dataset:
- Great work on the `Perceptron` class!
- I'm curious about the `Trainer` class, would be great to see it implemented. Seems to be some things like `computeGradients` and `costFunction`?
- How are you evaluating if your `NeuralNetwork` is doing what you expect? How would you make a prediction with it?

### Try building/training a more complex MLP on a bigger dataset.
- Nice work normalizing data and one-hot encoding targets. 😄 
- Instead of copying the `NeuralNetwork` code, consider modifying the `__init__` to take arguments for different input shapes, number of hidden nodes, etc.
- How many input nodes are needed for an image of size 28x28?
- A great challenge would be modifying this class to an output of 10 nodes with a softmax activation to work with this dataset!!
- As is, try it with the boston_housing data!
- Well done with all things Keras!
